docs/docs/features/partner-sharing.md

```diff
@@ -1,17 +1,61 @@
 # Partner Sharing
 
 Immich allows you to share your library with other users. They can then view your library and download the assets.
-
 You can manage one or multiple users to have access to your library from the [User Settings](docs/features/user-settings.md) page.
 
-<img src={require('./img/partner-sharing-1.png').default} title='Partner Sharing 1' />
+Partner sharing include:
+
+- All your photos and videos except those in Archived and Deleted.
+- You are at the mercy of the partners' actions (as it should be) on the asset they own. If the partner decides to delete or archive an asset belonging to them, it will not be displayed on your timeline.
+- The location where your photos were taken.
+- You can view partners' media and add them to an album. No other actions are allowed on those assets.
+- Facial data is not shared (yet)
```

Comment:
We can just leave out the yet part :D underpromise and overdeliver mindset please

Reply:
No problem, I just assumed it was a good idea to add it since there is a PR for it #7045.